packages/app/components/login/login-input-field.tsx

```diff
@@ -102,10 +90,10 @@ export function LoginInputField({
         onPress={handleSubmit(handleSubmitData)}
         variant="primary"
         size="regular"
-        tw="mt-6"
+        tw={`mt-6 ${!inputValue ? "opacity-50" : null}`}
```

Comment:
Nit: when composing styles conditionally, prefer to use an array of styles instead of
using string interpolation